tests/testlib/s2n_testlib.h

```diff
@@ -172,6 +172,9 @@ int s2n_read_test_pem_and_len(const char *pem_path, uint8_t *pem_out, uint32_t *
 int s2n_test_cert_chain_and_key_new(struct s2n_cert_chain_and_key **chain_and_key,
         const char *cert_chain_file, const char *private_key_file);
 
+S2N_RESULT s2n_test_cert_chain_from_pem(struct s2n_connection *conn, const char *pem_path,
+        struct s2n_stuffer *cert_chain_stuffer);
+
```

Comment:
My only real concern here is the naming. "cert_chain" usually refers to our s2n_cert_chain_and_key structure. See the method right above this one, `s2n_test_cert_chain_and_key_new`.

Since you're writing the contents of the Certificate message, maybe we should use the Certificate naming from the RFC? That would make this "s2n_test_certificate_list_from_pem". Alternatively, we could just specify that it's the message, not the struct: "s2n_test_cert_chain_data_from_pem" or ""s2n_test_cert_chain_message_from_pem".

Reply:
Ah right okay. I changed the name to `s2n_test_cert_chain_data_from_pem`.